R/utils.R

```diff
@@ -79,3 +79,38 @@ set_cond_number <- function(sample, k) {
 
   return(sample)
 }
+
+
+#' Minimal DAG from UG
+#' 
+#' find the DAG with same skeleton as the UG and 
+#' with the minimum number of v-structures added. 
+#' 
+#' @param x igraph graph or adjacency matrix
+#' @return adjacency matrix for directed graph
+#' @importFrom gRbase rip
```

Comment:
Same here, all these importFrom should not be needed (instead use gRbase::rip, etc.), since Matrix, gRbase and igraph are already imported in whole for the package